recipes/mosec/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cargo build --release
+$PYTHON -m pip install .
+mkdir -p $SP_DIR/$PKG_NAME/bin
+cp target/$CARGO_BUILD_TARGET/release/mosec $SP_DIR/$PKG_NAME/bin
+
+# generate the license file
+cargo install cargo-license
+cargo license --direct-deps-only --authors --avoid-build-deps --avoid-dev-deps --do-not-bundle --all-features --json > license.json
```

Comment:
```suggestion
cargo license --authors --avoid-build-deps --avoid-dev-deps --do-not-bundle --all-features --json > license.json
```
Indirect dependencies are still linked in the project, so they should be credited.

Reply:
Fixed. Please take a look.